examples/scaffold_example/server.py

```diff
@@ -53,14 +49,14 @@ def main(config: Dict[str, Any]) -> None:
         # We use the same fit config function, as nothing changes for eval
         on_evaluate_config_fn=fit_config_fn,
         evaluate_metrics_aggregation_fn=evaluate_metrics_aggregation_fn,
-        initial_parameters=initial_parameters,
-        initial_control_variates=initial_control_variates,
+        initial_parameters=get_initial_model_parameters(model),
+        model=model,
```

Comment:
Model is used by the SCAFFOLD strategy to automatically initialize the control variates to zero.